### DISCUSSION TITLE
No story files found for storybook@latest init

### DISCUSSION THREAD

**Discussion by josefvn** [storybookjs/storybook#26565] [Help]
### Summary

I'm getting errors from the Storybook **Lit + Vite (TS)** template out of the box:

> WARN No story files found for the specified pattern: src\**\*.mdx
> WARN No story files found for the specified pattern: src\**\*.stories.@(js|jsx|mjs|ts|tsx)

I created a new Storybook 8 project as follows (on Windows 11 using PowerShell):

```powershell
mkdir storybook
cd storybook
pnpm dlx storybook@latest init
```

When prompted to select a framework, I chose the **Lit + Vite (TS)** template. Everything installs just fine, storybook starts up, my browser opens to `http://localhost:6006`, and then I'm seeing the above mentioned errors.

Devtools console shows the following:

![Screenshot 2024-03-19 103524](https://github.com/storybookjs/storybook/assets/612695/b8ca19b9-b3e7-4380-bc36-9c3fa867954a)

The init CLI has created some sample stories in `src/stories` which I expect Storybook to pick up:

![Screenshot 2024-03-19 103757](https://github.com/storybookjs/storybook/assets/612695/61d475a0-67a1-47aa-8fe2-e07f4e195076)

The main.ts configuration also seems to be correct.

![Screenshot 2024-03-19 103851](https://github.com/storybookjs/storybook/assets/612695/9ee8bc75-384c-446b-a715-08f0696b25ea)

To reproduce, just follow my steps above. It is expected that Storybook would work with the sample stories generated by the init CLI.

I tried both NPM as well as PNPM. I also tried `npx storybook@latest init` with the **React + Vite (TS)** template.

### Additional information

package.json
```json
{
  "name": "storybook",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "preview": "vite preview",
    "storybook": "storybook dev -p 6006",
    "build-storybook": "storybook build"
  },
  "dependencies": {
    "lit": "^3.1.2"
  },
  "devDependencies": {
    "@chromatic-com/storybook": "1.2.23",
    "@storybook/addon-essentials": "^8.0.1",
    "@storybook/addon-links": "^8.0.1",
    "@storybook/blocks": "^8.0.1",
    "@storybook/test": "^8.0.1",
    "@storybook/web-components": "^8.0.1",
    "@storybook/web-components-vite": "^8.0.1",
    "storybook": "^8.0.1",
    "typescript": "^5.2.2",
    "vite": "^5.1.6"
  }
}
```

main.ts
```typescript
import type { StorybookConfig } from '@storybook/web-components-vite'

const config: StorybookConfig = {
  stories: [
    '../src/**/*.mdx',
    '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'
  ],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@chromatic-com/storybook'
  ],
  framework: {
    name: '@storybook/web-components-vite',
    options: {}
  },
  docs: {
    autodocs: 'tag'
  }
}
export default config
```

### Create a reproduction

To reproduce, simply run the following (Windows 11, PowerShell):

```powershell
mkdir storybook
cd storybook
pnpm dlx storybook@latest init
```

**Comment by monkeycatdog** [comment]
Got same issue. It's avoid somehow mdx files
